cuqi/distribution/_gmrf.py

```diff
@@ -147,7 +152,12 @@ def mean(self):
     
     @mean.setter
     def mean(self, value):
-        self._mean = force_ndarray(value, flatten=True)
+        # Force the mean to be an array of proper length to be used in RegularizedLinearRTO sampler.
```

Comment:
```suggestion
        # Force the mean to be an array of proper length.
```
I suggest not mentioning particular sampler here